jumpscale/packages/marketplace/chats/blog.py

```diff
@@ -7,17 +7,17 @@
 
 class BlogDeploy(Publisher):
 
-    title = "Deploy Blog"
-    SOLUTION_TYPE = "blog"  # chatflow used to deploy the solution
+    title = "Deploy a Blog"
+    SOLUTION_TYPE = "Blog"  # chatflow used to deploy the solution
```

Comment:
SOLUTION_TYPE is an enum, contains the types of the chatflows all lowered case shouldn't be changed(python semantics)